packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -7,6 +7,7 @@
  */
 
 import { ValuesType } from 'utility-types';
+import { ALERT_EVALUATION_VALUE } from './technical_field_names';
```

Comment:
I have the impression we don't need to import this at all, so no change at all in the default_alerts_as_data file. Unless I am missing something here. I was just thinking since other solutions already import ALERT_EVALUATION_VALUE from technical field map, it looks like it is already available to us, no? This part later on in the error rate panel is already enough, no?

```
import {
  ALERT_EVALUATION_VALUE,
} from '@kbn/rule-data-utils';
```

Reply:
CI removed the import, so we are good here!